packages/ui/src/proposals/modals/AddNewProposal/components/TriggerAndDiscussionStep.tsx

```diff
@@ -24,8 +24,11 @@ export const TriggerAndDiscussionStep = () => {
   ])
 
   const addMemberToWhitelist = (member: Member) => {
-    setValue('triggerAndDiscussion.discussionWhitelist', [...discussionWhitelist, member], { shouldValidate: true })
+    setValue('triggerAndDiscussion.discussionWhitelist', [...discussionWhitelist, member], {
+      shouldValidate: true,
+    })
   }
+
   const removeMemberFromWhitelist = (member: Member) => {
     setValue(
       'triggerAndDiscussion.discussionWhitelist',
```

Comment:
Wonder if it's worth to try something like
```suggestion
  const addMemberToWhitelist = (member: Member) => updateWhitelist([...discussionWhitelist, member])
  const removeMemberFromWhitelist = (member: Member) => updateWhitelist(discussionWhitelist.filter((m: Member) => m.id !== member.id))
  const updateWhitelist = (members: Member[]) => setValue('triggerAndDiscussion.discussionWhitelist', members, { shouldValidate: true })
```
(feel free to ignore)